### ISSUE TITLE
21388818: Developer Portal = WTF?

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#885]
#### Description

This is a duplicate of rdar://21372721

Whoever redesigned developer.apple.com doesn’t appear to understand how developers use it.

The marketing information presented on the page is useful only ONCE (to learn about the existence of new Apple technologies.)

The technical information on the page is a reference that gets used EVERY DAY by thousands of developers. And it’s terribly unusable.

We need to look up things like API diffs, release notes, sample code, downloads, certificates & provisioning: and all of these things are REALLY HARD to find in the new layout. There’s no consistency for where to find this information (some is in the body text, some is in the bottom navigation.)

And I’m still trying to figure out how to download iOS 9 so I can start testing it on my devices. You know, making all that marketing copy I just read about into a reality.

The marketing information is important, but don’t let it hide the technical information. Put the things developers use frequently all in one place.
## 

Product Version: N/A
Created: 2015-06-15T20:29:26.665390
Originated: 2015-06-15T21:29:00
Open Radar Link: http://www.openradar.me/21388818
